s3/deleter.go

```diff
@@ -29,3 +29,22 @@ func DeleteObject(client s3iface.S3API, bucket, file string) error {
 	_, err := client.DeleteObject(input)
 	return err
 }
+
+// DeleteObjects delete the keys in the specified bucket.
+func DeleteObjects(client s3iface.S3API, bucket string, files []string) error {
+	input := &s3.DeleteObjectsInput{
+		Bucket: aws.String(bucket),
+		Delete: &s3.Delete{
```

Comment:
how does s3.Delete behave if the input slice is empty?

Reply:
btw, that could be an additional test for deleter_test.go

Reply:
it doesn't delete anything nor raise an error: https://github.com/RedHatInsights/insights-operator-utils/pull/242/commits/320c11d3e54865b1ce253a695397e2864457f2e9